pool.go

```diff
@@ -358,6 +474,24 @@ func (p *pool) updatePoolWithNewWeightUnlocked(nm *Member, idx int) {
 	}
 }
 
+func (p *pool) isCoolOffLocked(node string) bool {
+	p.lk.Lock()
+	defer p.lk.Unlock()
+
+	oldVal := p.coolOffCount[node]
+	p.coolOffCount[node] = oldVal + 1
+
+	// reduce cool off duration if we've repeatedly seen a cool off request for this node.
+	newCoolOffMs := p.config.SaturnNodeCoolOff.Milliseconds() / int64(oldVal+1)
+	if newCoolOffMs == 0 {
+		newCoolOffMs = 1
```

Comment:
initially you had the smallest cool-off period as a minute. now it's one millisecond. is that where you want it?

Reply:
fixed this.